back/src/domains/core/naf/use-cases/GetNafSuggestions.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  NafSectionSuggestion,
+  SectionSuggestionsParams,
+  sectionSuggestionsParamsSchema,
+} from "shared";
+import { createTransactionalUseCase } from "../../UseCase";
+
+export type GetNafSuggestions = ReturnType<typeof makeGetNafSuggestions>;
+
+export const makeGetNafSuggestions = createTransactionalUseCase<
+  SectionSuggestionsParams,
+  NafSectionSuggestion[],
+  void,
+  void
+>(
+  {
+    inputSchema: sectionSuggestionsParamsSchema,
+    name: "NafSuggestions",
+  },
+  async ({ inputParams: { searchText }, uow }) => {
+    const sanitizedSearchText = searchText.toLowerCase();
```

Comment:
trim éventuellement ? 
```suggestion
    const sanitizedSearchText = searchText.toLowerCase().trim();
```